src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -6,6 +6,7 @@
 
 #include <array>
 #include <glad/glad.h>
+#include <thread>
```

Comment:
@JayFoxRox its only needed for std::this_thread which isn't used in the header, so it should go in the cpp file.